lib/api/CDataFrameAnalysisInstrumentation.cc

```diff
@@ -191,7 +191,8 @@ void CDataFrameAnalysisInstrumentation::monitor(CDataFrameAnalysisInstrumentatio
             instrumentation.flush();
             HANDLE_FATAL(<< "Input error: required memory "
                          << bytesToString(instrumentation.memory()) << " exceeds the memory limit "
-                         << bytesToString(instrumentation.m_MemoryLimit) << ". Please increase the limit to at least "
+                         << bytesToString(instrumentation.m_MemoryLimit)
+                         << ". Please force-stop the analysis job, increase the limit to at least "
```

Comment:
This reads slightly better to me:
```suggestion
                         << ". Please force-stop the analysis job and increase the limit to at least "
```

Reply:
Oh I didn't see the last part of the message: ignore.